src/api/app/models/linked_project.rb

```diff
@@ -28,7 +28,7 @@ def validate_duplicates
 #  linked_db_project_id       :integer          indexed => [db_project_id]
 #  position                   :integer
 #  linked_remote_project_name :string(255)
-#  vrevmode                   :string(8)        default("standard")
+#  vrevmode                   :string(8)        default("unextend")
```

Comment:
This looks like you forgot to add a migration, And would we need that change in 2.9 as well? Just recently [touched](#4648) this there...